src/Uno.UI.Runtime.Skia.Wpf/WPF/Extensions/CursorExtensions.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Windows.Input;
+using Windows.UI.Core;
+
+namespace Uno.UI.Skia.Platform.Extensions
+{
+	internal static class CursorExtensions
+	{
+		public static CoreCursor ToCoreCursor(this Cursor cursor)
+		{
+			CoreCursorType cursorType = CoreCursorType.Arrow;
+			if (cursor == Cursors.Wait)
```

Comment:
Feels like a good candidate for a switch expression ?

Reply:
I thought so, but the `Cursors.Xyz` are static properties. Is there a form of switch expression that can handle those elegantly?

Reply:
Right. Could be mixing up with C# 9.0 ;)